app/models/game.rb

```diff
@@ -2,6 +2,8 @@ class Game < ActiveRecord::Base
   has_many :pieces
   delegate :pawns, :knights, :bishops, :rooks, :queens, :kings, to: :pieces
 
+  attr_accessor :winner_id
```

Comment:
I can tell you that if you have a field in the db you definately don't want to have a attr_accessor by the same name as you will basically nullify the functionality.  So for sure get rid of that.  I'm not sure what you are having problems with, if you want to explain it more clearly i'd be happy to give you more advice.